mcs/class/corlib/ReferenceSources/Type.cs

```diff
@@ -34,10 +34,40 @@
 namespace System
 {
 	[Serializable]
+	[ComVisible (true)]
+	[ComDefaultInterfaceAttribute (typeof (_Type))]
```

Comment:
Should this line be guarded by `#if !MOBILE`?

Reply:
It's not guarded in case of Assembly (_Assembly), so I'm not sure it matters.

Reply:
It should be